modules/javafx.base/src/main/java/com/sun/javafx/binding/Logging.java

```diff
@@ -29,6 +29,8 @@
 
 public class Logging {
 
+    public static boolean keepException = false;
```

Comment:
I'd rather see a private field and a public setter/getter here. Please also add a comment on the setter/getter indicating that they are only used for testing.